packages/core/admin/admin/src/content-manager/pages/EditView/Information/index.js

```diff
@@ -58,7 +58,7 @@ const Body = () => {
   const getFieldInfo = (atField, byField) => {
     const user = initialData[byField] ?? {};
 
-    const displayName = getDisplayName(user, formatMessage);
+    const displayName = user ? getDisplayName(user, formatMessage) : '-';
```

Comment:
This is a bit odd, because user is `{}` as fallback as per the line above, so in theory it can never be falsy?